### DISCUSSION TITLE
What is "No Graph will run on TRT Execution provider" ?

### DISCUSSION THREAD

**Discussion by davodogster** [microsoft/onnxruntime#13935] [Other Q&A]
Is this warning bad or can it be safely ignored?

![image](https://user-images.githubusercontent.com/46079516/206935159-c7a114cc-6596-4a3f-b42e-8c4cfc056e3a.png)

**Comment by adrianlizarraga** [comment]
Hi @davodogster,

That warning indicates that TensorRT will not run any operators (i.e., graphs of operators) in your ONNX model. Instead, other execution providers, such as the default CPU execution provider or any other execution providers added to the session, will have to run the model.

This is a warning that should not be ignored if you encounter it _in your own application_ and you _expect_ the TensorRT execution provider to accelerate model inference.

However, I believe that your screenshot shows this warning occurring within the onnxruntime build tests. If this is the case, you can ignore the warnings as long as the tests pass.
